src/page-modules/departures/layout.tsx

```diff
@@ -40,39 +73,69 @@ function DeparturesLayout({ children }: DeparturesLayoutProps) {
 
   return (
     <div className={style.departuresPage}>
-      <div className={style.searchWrapper}>
-        <form className={style.searchContainer} onSubmit={onSubmitHandler}>
-          <div className={style.searchInput}>
-            <p className={style.searchInputLabel}>
+      <form className={style.container} onSubmit={onSubmitHandler}>
+        <div className={style.main}>
+          <div className={style.input}>
+            <Typo.p textType="body__primary--bold" className={style.heading}>
               {t(PageText.Departures.search.input.label)}
-            </p>
+            </Typo.p>
 
             <Search
               label={t(PageText.Departures.search.input.from)}
               onChange={setSelectedFeature}
             />
           </div>
 
-          <div className={style.searchDate}>
-            <p className={style.searchInputLabel}>
+          <div className={style.date}>
+            <Typo.p textType="body__primary--bold" className={style.heading}>
               {t(PageText.Departures.search.date.label)}
-            </p>
+            </Typo.p>
 
             <DepartureDateSelector
               initialState={departureDate}
               onChange={setDepartureDate}
             />
           </div>
+        </div>
+
+        <div className={style.buttons}>
```

Comment:
Is it possible to make these buttons be selected before the alternatives even though it precedes in order?

Reply:
Maybe you could use the FocusScope from react-aria to set focus when the showAlternatives button is clicked? This way the user doesn't have to tab through the "find departures" button?

Reply:
But I don't want to trap the focus within the alternatives, the user should be able to keep the alternatives open.

Reply:
That's configurable. I think we could do something like this (probably need some other tweaks but you get the idea)
```tsx
<FocusScope contain={false} restoreFocus autoFocus={showAlternatives}>
{showAlternatives && (...)}
</FocusScope>
```

Reply:
Added FocusScope and reordered the elements, so the order should now be correct.